### PR TITLE
0.6 - Update to 08/Dec/2016 - doc and Document Link requests

### PR DESCRIPTION
**Pull request by bruno-medeiros**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1473,6 +1484,47 @@ pub struct CodeLens {
...
+#[derive(Debug, PartialEq, Deserialize, Serialize)]
+pub struct DocumentLink {
+	/**
+	 * The range this link applies to.
+	 */
+	pub range: Range,
+	/**
+	 * The uri this link points to.
+	 */
+	pub target: String,
```
Comment (Marwes) [inline src/lib.rs]: Should this maybe be an `Url` instead of a `String`?

Comment (bruno-medeiros) [inline src/lib.rs]: Oops, definitely. Missed that one.

**Comment by bruno-medeiros** [comment]
Oops, I pushed the new change to master instead of the PR by mistake:
https://github.com/gluon-lang/languageserver-types/commit/cb639b84f6305f3bd77931affc05be8187ce6112

**Comment by bruno-medeiros** [comment]
BTW, as for moving to version 1.0.0, I realized it's probably best to leave that to when we can compile the crate with stable, instead of nightly.

**Pull request merged**